includes/data.php

```diff
@@ -778,6 +778,22 @@ function pods_v( $var = null, $type = 'get', $default = null, $strict = false, $
 					}
 				}
 				break;
+			case 'post_id':
+				if ( is_array( $var ) ) {
+					if ( isset( $var[ 'post_id' ] ) ) {
+						$post_id = $var[ 'post_id' ];
+					}
+					if ( isset( $var[ 'post_type' ] ) ) {
+						$post_type = $var[ 'post_type' ];
+					} else {
+						$post_type = 'post';
+					}
+				} else {
+					$post_id   = $var;
+					$post_type = 'post';
+				}
+				$output = apply_filters( 'wpml_object_id', $post_id, $post_type, true );
```

Comment:
This was specifically for WPML support.  I've not looked into the other translation plugins to see if they will require similar changes.  I was going to add some checks to only call the wpml_object_id filter if WPML is installed but figured since it was a filter there isn't any negative to it

I don't think adding an extra pods specific filter would be a bad thing, just don't know what someone would do with it.  Which is exactly the reason to add the filter :)